src/index.js

```diff
@@ -0,0 +1,41 @@
+const moment = require('moment-timezone');
```

Comment:
TODO: We don't strictly need to produce a browser build here, because bundlers configured like we do with Rollup can both

a) upconvert this CJS module to ES, albeit at the cost of treeshaking sometimes
b) resolve this `require` statement themselves, like we mark it as an `external` and a `global`

But would it be good practice to produce an ES build here nonetheless? Such as would guarantee that the upconversion was possible, and preemptively mark `moment-timezone` as an external.

Reply:
And if we do make a client-side build, is it then best practice to run the tests on that build too? I think all that would mean here is substituting the browser build of `moment-timezone` for the npm package i.e. we'd really just be testing that `moment-timezone` was isomorphic, as it's supposed to be.

Using the browser build of `moment-timezone` would probably mean testing using Erik rather than Jest which is kind of annoying, particularly insofar as Erik uses Jasmine with which I _think_ the Jest APIs are compatible, but not guaranteed to be so.

Reply:
I haven't taken a look everywhere through MixMax's shared modules but I'd be inclined to write the source in ES_ and transpile a browser build. I think that eliminates any need to flag anything as external or global, and as a consumer, my bundler should pick up the ES module if everything's configured right.

I wouldn't distinguish testing between client-server - I'd test the source code in Jest's node environment (as none of the code is specific to the BOM), and ideally use the same transpilation settings there. I've recently done this to our `react-common` module, configuring rollup with the latest babel [here](https://github.com/mixmaxhq/react-common/pull/422/commits/7f21e09693ba0f9529646ca227879333c476f9aa) and adding a `test` environment that Jest will automagically use in the babel config [here](https://github.com/mixmaxhq/react-common/pull/422/commits/defa10b02c082867b5a644a05a0c58a802c7ae60)

Reply:
> I'd be inclined to write the source in ES_ and transpile a browser build

Ah we could write the source as an ESM module yeah. If we did that we would then need to transpile a CJS build for use in Node.

I think we would still need to mark `moment-timezone` as `external` within this project's build process lest we hit [this](https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency).

`global` would be left to the application that consumes this library either way, to determine how the module is ultimately fulfilled. See "Bundling your own npm packages" [here](https://engineering.mixmax.com/blog/rollup-externals) for more info.

> I'd test the source code in Jest's node environment (as none of the code is specific to the BOM)

Yeah nothing's specific to the DOM (that's what you mean right?). What a client build would be testing would be:

1. That `moment-timezone`'s browser build works
2. That the interval would [continue running in the browser](https://github.com/mixmaxhq/day-start-interval/pull/1#discussion_r325838753) even if an invocation threw an exception.

Reply:
Decided not to test that the interval would continue running actually https://github.com/mixmaxhq/day-start-interval/pull/1/commits/73d2d1a3cbd546d8e83ae33f03bc843e9f3774c5.

Reply:
Descoped producing an ESM build for now https://github.com/mixmaxhq/day-start-interval/issues/4